src/utils/metrics.py

```diff
@@ -110,16 +138,43 @@ def __init__(self, labels):
                 ],
             )
         )
+        # Request stats
+        #   Latency
+        self.histogram_e2e_time_request = self.histogram_e2e_time_request_family.Metric(
```

Comment:
Do we need to think about a way to let the user choose what metrics they want to collect, or we just enforcing all of them?

Reply:
I think it's fine as long as no obvious performance drop. Users can use `curl localhost:8002/metrics | grep vllm:xxx` to gather metrics they need. Meanwhile I might start on the optimization after this PR.

Reply:
My $0.02, I think it's fine to start with the full set of vllm metrics we collect as binary on/off. We can consider enabling/disabling subsets upon future request or if we find significant issues with any particular metrics later on.